packages/repocop/src/utils.ts

```diff
@@ -1,5 +1,39 @@
-import type { Repository } from './types';
+import type { NonEmptyArray, Repository } from './types';
 
 export function isProduction(repo: Repository) {
 	return repo.topics.includes('production') && !repo.archived;
 }
+
+export function toNonEmptyArray<T>(value: T[]): NonEmptyArray<T> {
+	if (value.length === 0) {
+		throw new Error(`Expected a non-empty array. Source table may be empty.`);
+	}
+	return value as NonEmptyArray<T>;
+}
+
+//TODO test this
```

Comment:
There's tests in this PR right?

```suggestion
//TODO test this
```

Reply:
yes you're right!